backend/src/main/java/data/entities/CertificateType.java

```diff
@@ -1,19 +1,34 @@
 package data.entities;
 
+import com.fasterxml.jackson.annotation.JsonValue;
+
 public enum CertificateType {
 
     PERMISSION_OF_PRINTING("Permission of Printing"),
     CCC_CERTIFICATE("CCC certificate");
 
     private final String displayName;
+
     CertificateType(String displayName) {
         this.displayName = displayName;
     }
+
+    @JsonValue
     public String getDisplayName() {
```

Comment:
Oh no, in our world - code - this is enum and it is important to keep it enum in dto. In user world - GUI - this is some translated value. So, enum translations are added just as any other translation to your translations files.